scripts/fix_quiz_bias.py

```
@@ -0,0 +1,208 @@
+            continue
+        options = q.get("options", [])
+        correct = q.get("correct", -1)
+        if not is_biased(options, correct):
+            continue
+
+        print(f"  Fixing Q#{i}: {q.get('question', '')[:60]}...")
+        rewritten = rewrite_question(q, api_key)
+
+        # validate before overwriting
+        if not is_valid_question(rewritten, correct, len(options)):
+            print(f"    Validation failed — keeping original")
```

Ruff (0.15.18)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)